source/installation/index.rst

```diff
@@ -64,14 +64,37 @@ Python 3.10. Mantid v6.7 and before use Python 3.8.
 
 Latest Release
 ##############
+There are three ways to create and manage a Conda environment: using Mambaforge, Miniforge, or Anaconda.
+It is recommended to use Mambaforge as it's faster and better at resolving dependencies.
```

Comment:
I intended to leave the conda installation for backward compatibility. As some users use Conda and it is working fine, it is better to continue using it instead of installing Mamba because some conflicts happen when you install Mamba in an existing Conda installation.

Reply:
Maybe it could say something like "we recommend using mamba with the method below ... , but if you already have conda then you can use this alternative method ..."? Then it'll have both options, but it'll be clear that it's preferable to use mamba.